src/sentry/dynamic_sampling/rules/helpers/prioritize_transactions.py

```diff
@@ -0,0 +1,37 @@
+from typing import Mapping, Tuple, cast
+
+from sentry.dynamic_sampling.rules.utils import get_redis_client_for_ds
+from sentry.utils import json
+
+
+def _get_cache_key(org_id: int, proj_id: int) -> str:
+    return f"ds::o:{org_id}:p:{proj_id}:pri_tran"
```

Comment:
remember we decided to use a short version (without project_id):
https://github.com/getsentry/sentry/blob/41ff6d0ca205a37ce74d49780781136467a5f7d6/src/sentry/dynamic_sampling/rules/helpers/prioritise_project.py#L9-L10

and later use `hset` instead.